docs/running-tests.md

```
@@ -0,0 +1,62 @@
+npm ci
+npx playwright install --with-deps chromium
+
+export LANDINGPAGE_URL=$(./scripts/env-url.sh test2)
+npm test
+```
+
+`scripts/env-url.sh` reads the environment manifests in EduIDE-deployment, so
+the URL is never hardcoded here. Point `EDUIDE_DEPLOYMENT` at that checkout if
+it is not next door.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)